punycode.js

```diff
@@ -507,15 +507,20 @@
 	/** Expose `punycode` */
 	// Some AMD build optimizers, like r.js, check for specific condition patterns
 	// like the following:
-	if (
```

Comment:
We should keep the AMD condition as is for the AMD build optimizers (see the js comment above).